### PR TITLE
Pre-release merge for docs 0.6.2

### PR DESCRIPTION
**Pull request by bmaltzan**

### CODE REVIEW

**Review by jimentwood — APPROVED** [review]
looks good

File: deploy/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+### Build the image
+1. cd arxiv-docs
+1. export TAG=  ...Find a commit tag in git for the docker image tag.
+1. docker build . -t arxiv/labs:$TAG -f ./Dockerfile-labs
+1. docker push arxiv/labs:$TAG
+1. docker run -it -p 8000:8000 arxiv/labs:$TAG
+
+### Deploy the image
+1. ssh arxiv-bastion
+1. helm ls | grep labs-static
```
Comment (bdc34) [inline deploy/README.md:14]: Just curious, what is the purpose of this line?

**Review by bdc34 — APPROVED** [review]
Looks fine to me.

File: deploy/README.md (hunk 1 above)
Comment (bmaltzan) [reply deploy/README.md:14]: Just for developer sanity. Ahead of changes, helm-ls shows me the current state: that it's running, current revision, and last update date.

File: labs/index.md [hunk 2]
```diff
@@ -40,7 +75,8 @@ While the arXiv team is focused on our core mission—providing rapid disseminat
 
 Here are some of the projects that our collaborators are working on right now.
 
+$jinja {{ render_project(projects.pwc-links) }} jinja$
```
Comment (bmaltzan) [inline labs/index.md:78]: typo: -  -->  _

File: deploy/README.md (hunk 1 above)
Comment (bdc34) [reply deploy/README.md:14]: Thanks.

**Pull request merged**